circulating_supply/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "circulating_supply"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix = "0.13.0"
+anyhow = "1.0.51"
+bigdecimal = "=0.1.0"
+base64 = "0.11"
+chrono = "0.4.19"
+dotenv = "0.15.0"
+# syn version conflict, replace with crates.io version once released
+near-sdk = { git = "https://github.com/near/near-sdk-rs", rev="03487c184d37b0382dd9bd41c57466acad58fc1f" }
+tokio = { version = "1.1", features = ["sync", "time"] }
+tracing = "0.1.13"
+tracing-subscriber = "0.2.4"
+uint = { version = "0.8.3", default-features = false }
+
+near-indexer-primitives = "0.14"
```

Comment:
Any reason for not using the latest `0.15` version?

Reply:
this question applies also on `near-jsonrpc-primitives`

Reply:
The latest version of `near-jsonrpc-client-rs` uses these versions under the hood. Using `0.15` causes `mismatched types` when using them together

Reply:
near-jsonrpc-client-rs 0.4.0 (release) uses 0.15 versions:

https://github.com/near/near-jsonrpc-client-rs/blob/master/Cargo.toml#L27-L30

Can you try upgrading all the dependencies and report what is not matching?

Reply:
`explorer-database` (the indexer database logic which has been extracted to a library crate) uses `near-primitives:0.14`, so using `0.15` in `circulating_supply` would cause conflicts (`lockup_account_ids` are fetched from the DB and feed to JSON RPC).

I'm not sure upgrading `explorer-database` to `near-primitives:0.15` is feasible as this would require `indexer` to also use `0.15`, and the latest `near-lake-framework` [isn't compatible with this version](https://github.com/near/near-lake-framework-rs/blob/main/Cargo.toml#L30).

Reply:
I see. Well, we should be fine going with 0.14 for now, and then upgrade all the crates at once after the framework is upgraded. I created a tracking issue https://github.com/near/near-lake-framework-rs/issues/48